src/world/area_kkj/kkj_14/ADAEA0.c

```diff
@@ -1,3 +1,10 @@
 #include "kkj_14.h"
 
-INCLUDE_ASM(s32, "world/area_kkj/kkj_14/ADAEA0", func_80240000_ADAEA0);
+extern char D_800D9668[];
+
+ApiStatus func_80240000_ADAEA0(Evt* script, s32 isInitialCall) {
+    if (evt_get_variable(NULL, GSW(0)) == -128) {
+        sprintf(D_800D9668, "nok_bg\0\0pra_01\0\0kkj_11\0\0kkj_01\0\0kkj_26\0\0kkj_15\0\0kkj_27\0\0kkj_26");
```

Comment:
technically this should just be nok_bg and there should be some other static char arrays (or one big one for now) for the rest of these strings. you could place it directly after this function.

Reply:
I'm assuming you're talking about those `static char* N(exit_str_x)` strings, is that right? I tried to use this pattern in a couple of ways. The function matches but the produced assembly fails somewhere. I think I just don't have a good grasp of how things are connected yet. Are you suggesting something like this:
```c
ApiStatus func_80240000_ADAEA0(Evt* script, s32 isInitialCall) {
    if (evt_get_variable(NULL, GSW(0)) == -128) {
        sprintf(D_800D9668, "nok_bg");
    }
    return ApiStatus_BLOCK;
}

static char* N(exit_str_0) = "\0pra_01\0\0kkj_11\0\0kkj_01\0\0kkj_26\0\0kkj_15\0\0kkj_27\0\0kkj_26";
```

Reply:
yeah, exactly that! I recommend using vbindiff when solving issues like these, but it can be tricky. I'm happy to try looking on your branch as well if you want

Reply:
ah, I just looked and you have it correct - you just need to remove the leading \0 from your string below the function, and it matches :) we can merge once this is done

Reply:
Thanks for looking! I was confident I had tested this possibility. PR ready to be merged!